<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>AKbalthom Freehand</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Speed : </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="speedIndicator">
         <property name="font">
          <font>
           <family>AKbalthom HighSchool-Frame</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>AKbalthom HighSchool-Frame</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Km/H</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="levelIndicator">
           <property name="font">
            <font>
             <family>AKbalthom HighSchool-Frame</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Level N</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>AKbalthom Freehand</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Remaining Distance : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="RemainingDistanceIndicator">
           <property name="font">
            <font>
             <family>AKbalthom HighSchool-Frame</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>AKbalthom HighSchool-Frame</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>m</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
